lib/gzip-static.js

```diff
@@ -21,17 +21,17 @@ function setHeader(res, path, encoding) {
  */
 async function createCaches(root, methods) {
   const dirents = await fs.readdir(root, {
-    recursive: true,
-    withFileTypes: true
+    recursive: true, 
+    withFileTypes: true 
   });
   dirents.forEach(dirent => {
     if (!dirent.isFile()) {
       return;
     }
-    const { name, parentPath } = dirent;
+    const { name, path: relativePath } = dirent;
```

Comment:
isn't `path` a deprecated property replaced by `parentPath` - what are we really changing here?